Package.swift

```diff
@@ -80,6 +80,10 @@ let package = Package(
         // restore original package url once https://github.com/wickwirew/Runtime/pull/93
         // and https://github.com/wickwirew/Runtime/pull/95 are merged
         // .package(url: "https://github.com/wickwirew/Runtime.git", from: "2.2.3"),
+        
+        // ApodiniTypeInformation
+        .package(url: "https://github.com/Apodini/ApodiniTypeInformation.git", from: "0.2.0"),
```

Comment:
We might want to use `.upToNextMinor(from:)` to not automatically jump to 0.3.0, which might contain breaking changes.